test/util/testnode/config.go

```diff
@@ -25,7 +25,10 @@ const (
 	DefaultValidatorAccountName = "validator"
 	DefaultInitialBalance       = genesis.DefaultInitialBalance
 	// TimeoutCommit is a flag that can be used to override the timeout_commit.
+	// Deprecated: Use BlockTimeFlag instead.
 	TimeoutCommitFlag = "timeout-commit"
+	// BlockTimeFlag is a flag that can be used to override the DelayedPrecommitTimeout.
+	BlockTimeFlag = "block-time"
```

Comment:
this is different in celestia-app. I think `block-time` was renamed to something else.

Reply:
good catch https://github.com/celestiaorg/celestia-app/pull/5639/commits/1c8cd007ebd004eb449f2385693a4d550f2049f7